unity-renderer/Assets/ABConverter/Utils.cs

```diff
@@ -16,6 +16,27 @@
 
 namespace DCL.ABConverter
 {
+    public static class MeshUtils
```

Comment:
Duplicated on purpose so the ABConverter doesn't have any reference from our main project.